### ISSUE TITLE
Azure DevOps publish fails: Error: template: pattern matches no files: `templates\markdown-html.tmpl`

### ISSUE THREAD

**Issue by yoogie27** [infracost/infracost#2518]
We are using version 0.10.23 since yesterday in our Azure DevOps pipeline.
We are running multiple plans, uploading them as pipeline artifacts and then use a separate stage to produce the cost diff and publish to the PR.

This used to work fine until yesterday, when you have released a new version.

The error message is:
```
Error: template: pattern matches no files: `templates\markdown-html.tmpl`
```

The relevant pipeline command is:
```
  # publish cost difference to the azure pull request
  - task: PowerShell@2
    continueOnError: true
    inputs:
      targetType: 'inline'
      script: |
        infracost comment azure-repos --path=diff-infracost-1.json --path=diff-infracost-2.json --path=diff-infracost-3.json --path=diff-infracost-4.json --show-all-projects --azure-access-token=$(System.AccessToken) --pull-request=$(System.PullRequest.PullRequestId) --repo-url=$(Build.Repository.Uri) --behavior=update
      workingDirectory: $(System.DefaultWorkingDirectory)/repo/${{ parameters.terraformFolder }}
    displayName: Post Infracost Comment
```

**Comment by vdmgolub** [comment]
@yoogie27 Hello! Thank you for creating the issue! We're looking into this and will notify you when the fix is released.

**Comment by aliscott** [comment]
Hi @yoogie27, thanks for reporting this 🙏 . A fix has now been released in v0.10.24. Can you confirm that this is now working okay for you?

**Comment by yoogie27** [comment]
> Hi @yoogie27, thanks for reporting this 🙏 . A fix has now been released in v0.10.24. Can you confirm that this is now working okay for you?

It works now. Many thanks for your quick turnaround
